delivery_postlogistics/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# © 2013-2016 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Postlogistics Shipping",
```

Comment:
```suggestion
PostLogistics Shipping - “Barcode” web service
```

https://geschaeftsbericht.post.ch/17/ar/en/category/post_logistics_en/

PostLogistics is written with a capital `L`

And better to name it after the official name so users can find it quickly.
https://www.post.ch/en/post-barcode-handbuch.pdf
https://www.post.ch/en/sending-letters/franking-mail/franking-solutions-for-letters/support-for-consignments-with-barcode/sending-consignments-web-service-barcode